### PR TITLE
Remove debug log messages from kama trend calculator.

### PR DESCRIPTION
**Pull request by HouzuoGuo**
Remove extraneous debug log messages from kama trend calculator.

### CODE REVIEW

**Comment by HouzuoGuo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by cinar — APPROVED** [review]
Good catch! Thank you for your contribution!

**Comment by cinar** [comment]
Could you remove the log import? Also, if you can run ./pre-commit.sh, it should take care of cleaning the unused imports and formatting the files. Many thanks!

**Comment by HouzuoGuo** [comment]
... that was embarrassing, let me try again using a proper editor instead of the web based shortcut.
